x-pack/metricbeat/module/azure/billing/client_test.go

```diff
@@ -7,12 +7,15 @@ package billing
 import (
 	"errors"
 	"testing"
+	"time"
 
-	prevConsumption "github.com/Azure/azure-sdk-for-go/services/consumption/mgmt/2019-01-01/consumption"
-	"github.com/Azure/azure-sdk-for-go/services/consumption/mgmt/2019-10-01/consumption"
+	// consumption "github.com/Azure/azure-sdk-for-go/services/consumption/mgmt/2019-01-01/consumption"
+	// "github.com/Azure/azure-sdk-for-go/services/consumption/mgmt/2019-10-01/consumption"
```

Comment:
Should not these commented lines be removed?

```suggestion
```

Reply:
Definitely. Thanks! 🙇